language/move-lang/stdlib/modules/libra_coin.move

```diff
@@ -4,14 +4,19 @@ module LibraCoin {
     use 0x0::Transaction;
 
     // A resource representing the Libra coin
-    // The value of the coin. May be zero
-    resource struct T { value: u64 }
+    resource struct T {
+        // The value of the coin. May be zero
+        value: u64,
+    }
 
     // A singleton resource that grants access to `LibraCoin::mint`. Only the Association has one.
-    resource struct MintCapability {}
+    resource struct MintCapability { }
 
-    // The sum of the values of all LibraCoin::T resources in the system
-    resource struct MarketCap { total_value: u64 }
+    resource struct MarketCap {
+        // The sum of the values of all LibraCoin::T resources in the system
+        //total_value: u128,
+        total_value: u64,
```

Comment:
FIXME?

Reply:
Yes. But soon not to be now that @tnowacki has landed them in the source language! I'll update now :)